plutus-core/executables/uplc/Main.hs

```diff
@@ -67,6 +68,7 @@ data EvalOptions =
       Input
       Format
       PrintMode
+      Bool -- Use de Bruijn indices in the output?
```

Comment:
I personally think it's better to write a new datatype with 2 constructors something like  `ShowNames = UseDeBruijn | UseNames` to avoid "boolean blindness"; it makes it possible to figure out from just the type/constructors what semantics they have, if you use booleans you have to add comments to maintain readability.

Reply:
You caught me!  I did think about doing that but was too lazy, partly because I would have had to go and look up the right way to deal with the command-line options.  I'll change it.